src/app/core/data/data.service.ts

```diff
@@ -42,7 +41,7 @@ export abstract class DataService<T extends CacheableObject> {
   protected abstract requestService: RequestService;
   protected abstract rdbService: RemoteDataBuildService;
   protected abstract dataBuildService: NormalizedObjectBuildService;
-  protected abstract store: Store<CoreState>;
+  protected abstract store: Store<any>;
```

Comment:
Why not AppState?

Reply:
I moved BitstreamFormatRegistryState from AppState to CoreState, so now it's reverted to CoreState